tests/ops/test_qubit_ops.py

```diff
@@ -151,31 +151,6 @@ def test_hermitian_eigegendecomposition_multiple_wires(
         assert np.allclose(qml.Hermitian._eigs[key]["eigvec"], eigvecs, atol=tol, rtol=0)
         assert len(qml.Hermitian._eigs) == 1
 
-    @pytest.mark.parametrize("observable", EIGVALS_TEST_DATA_MULTI_WIRES)
-    def test_hermitian_eigegendecomposition_multiple_wires_repeat_with_different_wires_new_entry(
-        self, observable, tol
-    ):
-        """Tests that the eigendecomposition_of_permuted property of the Hermitian class returns the correct results
-        for multiple wires when the same operator is specified twice, but with different wires."""
-
-        num_wires = int(np.log2(len(observable)))
-        eigendecomp = qml.Hermitian(observable, wires=list(range(num_wires))).eigendecomposition_of_permuted
-
-        eigvals, eigvecs = np.linalg.eigh(observable)
-
-        assert np.allclose(eigendecomp["eigval"], eigvals, atol=tol, rtol=0)
-        assert np.allclose(eigendecomp["eigvec"], eigvecs, atol=tol, rtol=0)
-
-        key = tuple(observable.flatten().tolist())
-        assert np.allclose(qml.Hermitian._eigs[key]["eigval"], eigvals, atol=tol, rtol=0)
-        assert np.allclose(qml.Hermitian._eigs[key]["eigvec"], eigvecs, atol=tol, rtol=0)
-        assert len(qml.Hermitian._eigs) == 1
-
-        # Reverse the order of the wires to check if a new entry was added
-        eigendecomp = qml.Hermitian(observable, wires=list(range(num_wires))[::-1]).eigendecomposition_of_permuted
-
-        assert len(qml.Hermitian._eigs) == 2
```

Comment:
This test is no longer required, as the permutation no longer occurs inside `Hermitian.eigvals`. As a result, calling Hermitian with different wire orderings will no longer create a new key in the matrix!